src/Tizen/Tizen.Common/IWindowProvider.cs

```diff
@@ -1,9 +1,25 @@
-﻿using System;
+﻿/*
+* Copyright (c) 2024 Samsung Electronics Co., Ltd All Rights Reserved
```

Comment:
if the file is moved, then maybe it would make sense to have the year it was originally created

Reply:
got it. no change would be done.